code/index.html

```diff
@@ -1,30 +1,181 @@
+      <a href="#">about us</a>
+      <a href="#">news</a>
+      <a href="#">cafes</a>
+    </nav>
+    <div>
+      <button class="sign-in">Sign in</button>
+    </div>
+  </div>
+  <!--responsive image-->
+  
+  <picture>
+    <source media="(max-width: 678px)" srcset="./images/1000.jpg">
+    <source media="(max-width: 1024px)" srcset="./images/1200.jpg">
+    <img src="./images/origin.jpg" alt="Beautiful coffee maker picture">
+  </picture>
+
 
-      <div class="question" id="section2">Section 2</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+</header>
+
+<!--main section-->
+  <div class="main">
+    <!-- signup form -->
+    <div class="signup">
+      <legend>Please sign up here:</legend>
+      <form action="http://httpbin.org/anything" method="POST">
+        <fieldset>
+          <div class="input_data">
+            <label>
+              Name:
+              <input name="name" type="text" class="form-group" placeholder="Type your last name" required>
+            </label>
+            <label>
+              Email:
+              <input name="email" type="email" class="form-group" placeholder="Type your email" required />
+            </label>
+            <label>
+              Password:
+              <input name="password" type="password" class="form-group" placeholder="Type your password" required />
+            </label>
+            <label>
+          </div>
+          <div class="selection">
+            Who are you?
+            <select name="coffee_fan">
+              <option value="snob">coffee snob</option>
+              <option value="enthusiast">intermediate enthusiast</option>
+              <option value="noob">n00b</option>
+            </select>
+            </label>
+          </div>
+    
+          <div class="radio-buttons">
+            <p>Please select your gender:</p>
+            <input type="radio" id="male" name="gender" value="male" />
+            <label for="male">Male</label><br />
+            <input type="radio" id="female" name="gender" value="female" />
+            <label for="female">Female</label><br />
+            <input type="radio" id="other" name="gender" value="other" />
+            <label for="other">Other</label>
+          </div>
+    
+          <div class="checkbox">
+            <input type="checkbox" id="input_data_checkbox" name="input_data" />
+            <label for="checkbox">I agree to get cafeinated</label>
+          </div>
+          <div class="submit_form">
+            <button class="submit" value="SUBMIT">SUBMIT</button>
+          </div>
+        </fieldset>
+      </form>
+    </div>
+    
+    <!-- accordion -->
+    <div class="accordion">
+      <div class="accordion-intro">
+        <h4>Wanna learn something about coffee today?</h4>
+      </div>
+      <div class="accordion-container">
+        <h3 id="accordion-header-1" class="accordion-header">
+          <span>Who discovered coffee?</span>
+          <span>▼</span>
+        </h3>
```

Comment:
The text doesn't need to be in a `<span>` since it's wrapped in a heading, and the span isn't styled but the tiny arrow should be.